kontrollsamtale/infrastructure/src/main/kotlin/no/nav/su/se/bakover/kontrollsamtale/infrastructure/web/KontrollsamtaleRoutes.kt

```diff
@@ -71,4 +73,14 @@ fun Route.kontrollsamtaleRoutes(
             }
         }
     }
+
+    get("kontrollsamtaler/{sakId}") {
```

Comment:
Dette er vel ikke standard URL-oppbygning for oss? Den bruker typ være /sak/{sakId}/kontrollsamtaler